containers/ddev-php-base/generic-files/usr/local/bin/build_php_extension.sh

```diff
@@ -2,8 +2,8 @@
 
 set -eu -o pipefail
 
-if [ "$#" -ne 4 ]; then
-  echo "Usage: $0 <PHP_VERSION> <EXTENSION_NAME> <EXTENSION_VERSION> <EXTENSION_FILE>"
+if [ "$#" -ne 4 ] && [ "$#" -ne 5 ] && [ "$#" -ne 6 ]; then
```

Comment:
nit: This might be clearer with something like reversing the logic and saying between, or something like that. I had to look at it for a while!